airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationRunner.java

```diff
@@ -68,13 +71,26 @@ public IntegrationRunner(Source source) {
                     Consumer<AirbyteMessage> outputRecordCollector,
                     Destination destination,
                     Source source) {
-    Preconditions.checkState(destination != null ^ source != null, "can only pass in a destination or a source");
+    Preconditions.checkState(destination != null ^ source != null,
+        "can only pass in a destination or a source");
     this.cliParser = cliParser;
     this.outputRecordCollector = outputRecordCollector;
     // integration iface covers the commands that are the same for both source and destination.
     this.integration = source != null ? source : destination;
     this.source = source;
     this.destination = destination;
+    validator = new JsonSchemaValidator();
+  }
+
+  @VisibleForTesting
+  IntegrationRunner(IntegrationCliParser cliParser,
+                    Consumer<AirbyteMessage> outputRecordCollector,
+                    Destination destination,
+                    Source source,
+                    JsonSchemaValidator jsonSchemaValidator) {
+
```

Comment:
Nit. This extra line seems unnecessary.

Reply:
Ok, I will update it. I just have the plugin enabled that automatically formats code by "google style" for java code. So assume that long lines were splitting by this plugin.
Thanks